roles/lib_utils/tasks/main.yml

```diff
@@ -8,4 +8,6 @@
   package:
     name: python-ruamel-yaml
```

Comment:
This says python-ruamel-yaml.   I placed python2-ruamel-yaml in lib_openshift.  Should they be the same? I know that python2-ruamel versione exists.  The other does not, yet.

I'm out of the loop on what the official one will be.